### ISSUE TITLE
Error: Haplotype is too short for alignment

### ISSUE THREAD

**Issue by DiDeoxy** [luntergroup/octopus#133]
**Describe the bug**
Error: Haplotype is too short for alignment

**Version**

```shell
$ octopus --version
octopus version 0.7.0 (develop 302c444d)
Target: x86_64 Linux 3.10.0-1127.8.2.el7.x86_64
SIMD extension: AVX2
Compiler: GNU 10.2.0
Boost: 1_73
```

**Command**
Command line to install octopus:
```shell
$ ./octopus/scripts/install.py --prefix ~/bin -D
```

Command line to run octopus:
```shell
$ octopus \
        -R $reference \
        -I $bam \
        -T chr1H chr2H chr3H chr4H chr5H chr6H chr7H \
        --refcall \
        --debug \
        --sequence-error-model PCR.HiSeq-2500 \
        -o ${sample_id}.g.vcf
```

**Additional context**
Add any other context about the problem here, e.g.

 - Reference [e.g. hg19]. Please provide link to fasta if atypical.
       - https://plants.ensembl.org/Hordeum_vulgare/Info/Index

[octopus_debug.zip](https://github.com/luntergroup/octopus/files/5202051/octopus_debug.zip)

 - BAM files (if possible).

**Comment by dancooke** [comment]
Thanks for the bug report. Are you able to share the BAM file? The error occurs around `chr3H:449833583` so just a section around there should be enough to reproduce.

**Comment by DiDeoxy** [comment]
[error_region_ERR2221531.zip](https://github.com/luntergroup/octopus/files/5202385/error_region_ERR2221531.zip)

I 2Mb region centered on the spot, let me know is you need more.

**Comment by dancooke** [comment]
Thanks. Should be fixed in d300da7659e50700cf4a68ddbc4c6d1cd836aa62. Please re-open if any further problems.
